projects/experimental/components/badge/badge.component.ts

```diff
@@ -0,0 +1,44 @@
+import {ChangeDetectionStrategy, Component, Inject, Input} from '@angular/core';
+import {TUI_PLATFORM, TuiPlatform} from '@taiga-ui/cdk';
+import {
+    MODE_PROVIDER,
+    TUI_MODE,
+    TuiBrightness,
+    tuiPop,
+    TuiSizeS,
+    TuiSizeXL,
+} from '@taiga-ui/core';
+import {TuiStatus} from '@taiga-ui/kit';
+import {Observable} from 'rxjs';
+
+@Component({
+    selector: 'tui-badge',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    templateUrl: './badge.template.html',
+    styleUrls: ['./badge.style.less'],
+    providers: [MODE_PROVIDER],
+    animations: [tuiPop],
+    host: {
+        '[attr.data-platform]': 'platform',
+        '[attr.data-appearance]': 'appearance',
+        '[attr.data-size]': 'size',
+        '[class._hoverable]': 'hoverable',
+        '[tabIndex]': 'hoverable ? 0 : -1',
```

Comment:
If hoverable is mostly legacy or hint showing, let's not make it focusable? We can also remove focus styles.